.circleci/config.yml

```diff
@@ -17,6 +17,22 @@ references:
         pacman -Syu --noconfirm python python-pip git base-devel curl
         pip install coveralls coverage
 
+  ubuntu_deps_opencv: &ubuntu_deps_opencv
+    run:
+      name: Install opencv dependencies on Ubuntu.
+      command: |
+        export DEBIAN_FRONTEND=noninteractive
+        apt-get install -y --force-yes opencv-data libopencv-dev python3-opencv
```

Comment:
Do we need `opencv-data` ?

Reply:
I copied this line from blocks config, as above, I'll try without it and we'll see